Automattic/wp-calypso
# Components: Cleanup unused `LanguagePicker` functionality

#### Changes proposed in this Pull Request

This PR removes a few unused language picker utils and constants:

* `getLanguageGroupById()` - unused since #48374
* `getLanguageGroupFromTerritoryId()` - #20815 - unused since its creation
* `getLanguageGroupByLangSlug()` - #20815 - unused since its creation
* `getLanguageGroupByCountryCode()` - unused since #48374
* `<LanguagePickerItemTooltip />` - unused since #48374

#### Testing instructions

* Verify the removed code is not in use
* Verify Calypso builds and the language picker still works well.
